### ISSUE TITLE
Why my Classification modelnet40 get so low accuracy，請問為何我的分類正確率特別低?

### ISSUE THREAD

**Issue by firecoolman** [yanx27/Pointnet_Pointnet2_pytorch#217]
2023-01-19 22:17:58,794 - Model - INFO - PARAMETER ...
2023-01-19 22:17:58,794 - Model - INFO - Namespace(batch_size=24, decay_rate=0.0001, epoch=200, gpu='0', learning_rate=0.001, log_dir='pointnet2_cls_ssg2', model='pointnet2_cls_ssg', num_category=40, num_point=1024, optimizer='Adam', process_data=False, use_cpu=False, use_normals=False, use_uniform_sample=False)
2023-01-19 22:17:58,794 - Model - INFO - Load dataset ...
2023-01-19 22:37:44,447 - Model - INFO - No existing model, starting training from scratch...
2023-01-19 22:37:44,448 - Model - INFO - Start training...
2023-01-19 22:37:44,448 - Model - INFO - Epoch 1 (1/200):
2023-01-19 22:54:20,687 - Model - INFO - Train Instance Accuracy: 0.078354
2023-01-19 22:54:55,380 - Model - INFO - Test Instance Accuracy: 0.000000, Class Accuracy: 0.000000
2023-01-19 22:54:55,380 - Model - INFO - Best Instance Accuracy: 0.000000, Class Accuracy: 0.000000
2023-01-19 22:54:55,380 - Model - INFO - Save model...
2023-01-19 22:54:55,380 - Model - INFO - Saving at log\classification\pointnet2_cls_ssg2\checkpoints/best_model.pth
2023-01-19 22:54:55,427 - Model - INFO - Epoch 2 (2/200):
2023-01-19 22:57:17,406 - Model - INFO - Train Instance Accuracy: 0.086484
2023-01-19 22:57:50,877 - Model - INFO - Test Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 22:57:50,879 - Model - INFO - Best Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 22:57:50,879 - Model - INFO - Save model...
2023-01-19 22:57:50,879 - Model - INFO - Saving at log\classification\pointnet2_cls_ssg2\checkpoints/best_model.pth
2023-01-19 22:57:50,930 - Model - INFO - Epoch 3 (3/200):
2023-01-19 23:00:08,223 - Model - INFO - Train Instance Accuracy: 0.086585
2023-01-19 23:00:41,749 - Model - INFO - Test Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:00:41,749 - Model - INFO - Best Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:00:41,750 - Model - INFO - Save model...
2023-01-19 23:00:41,750 - Model - INFO - Saving at log\classification\pointnet2_cls_ssg2\checkpoints/best_model.pth
2023-01-19 23:00:41,790 - Model - INFO - Epoch 4 (4/200):
2023-01-19 23:03:01,451 - Model - INFO - Train Instance Accuracy: 0.087093
2023-01-19 23:03:34,224 - Model - INFO - Test Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:03:34,224 - Model - INFO - Best Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:03:34,224 - Model - INFO - Save model...
2023-01-19 23:03:34,224 - Model - INFO - Saving at log\classification\pointnet2_cls_ssg2\checkpoints/best_model.pth
2023-01-19 23:03:34,263 - Model - INFO - Epoch 5 (5/200):
2023-01-19 23:05:59,614 - Model - INFO - Train Instance Accuracy: 0.088821
2023-01-19 23:06:36,431 - Model - INFO - Test Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:06:36,431 - Model - INFO - Best Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:06:36,432 - Model - INFO - Save model...
2023-01-19 23:06:36,432 - Model - INFO - Saving at log\classification\pointnet2_cls_ssg2\checkpoints/best_model.pth
2023-01-19 23:06:36,467 - Model - INFO - Epoch 6 (6/200):
2023-01-19 23:09:09,582 - Model - INFO - Train Instance Accuracy: 0.087398
2023-01-19 23:09:45,079 - Model - INFO - Test Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:09:45,079 - Model - INFO - Best Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:09:45,079 - Model - INFO - Save model...
2023-01-19 23:09:45,079 - Model - INFO - Saving at log\classification\pointnet2_cls_ssg2\checkpoints/best_model.pth
2023-01-19 23:09:45,111 - Model - INFO - Epoch 7 (7/200):
2023-01-19 23:12:00,355 - Model - INFO - Train Instance Accuracy: 0.088516
2023-01-19 23:12:33,710 - Model - INFO - Test Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:12:33,710 - Model - INFO - Best Instance Accuracy: 0.040453, Class Accuracy: 0.025000
2023-01-19 23:12:33,711 - Model - INFO - Save model...
2023-01-19 23:12:33,711 - Model - INFO - Saving at log\classification\pointnet2_cls_ssg2\checkpoints/best_model.pth
2023-01-20 09:27:24,303 - Model - INFO - Epoch 99 (99/200):
Train Instance Accuracy: 0.794715
Test Instance Accuracy: 0.040453, Class Accuracy: 0.025000
Best Instance Accuracy: 0.040453, Class Accuracy: 0.025000

when i train to 99 epoch ,my accuracy still very low .What did I do wrong or what do I need to improve?I am truly grateful for someone tell me.Thank you

**Comment by cesaryuan** [comment]
Hi, I have the same issue as you. Have you figured it out?

**Comment by firecoolman** [comment]
> Hi, I have the same issue as you. Have you figured it out?

I choose another github to train my own data to achieve classfication 3D model 
https://github.com/zhulf0804/Pointnet2.PyTorch
hope this github can help you

**Comment by ludashi451** [comment]
我是在Windows平台上出现了这样的情况，我在乌班图系统上没有出现这种情况

**Comment by lovepythonhm** [comment]
> > Hi, I have the same problem as you. Have you figured it out?
> 
> I chose another github to train my own data to achieve a classified 3D model https://github.com/zhulf0804/Pointnet2.PyTorch hope this github can help you
Is this problem solved on the win system now?

**Comment by lovepythonhm** [comment]
> Is this problem solved on the win system now?

**Comment by lovepythonhm** [comment]
> Hi, I have the same problem as you. Have you figured it out?

Is this problem solved on the win system now?

**Comment by lovepythonhm** [comment]
加Q交流：2287417720

**Comment by yuanlinwang** [comment]
same problem occurred, is it due to the windows OS?

**Comment by lovepythonhm** [comment]
No, add me qq: 2287417720 detailed exchange




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yanx27/Pointnet_Pointnet2_pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月25日(星期天) 晚上8:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yanx27/Pointnet_Pointnet2_pytorch] Why my Classification modelnet40 get so low accuracy，請問為何我的分類正確率特別低? (Issue #217)





 
same problem occurred, is it due to the windows OS?

**Comment by liu-jun-chen** [comment]
Met the same problem on Windows 11. Does anyone know where the problem is?

**Comment by USTBVEO** [comment]
same problem. I also used the windows os.

**Comment by UCASAlbert** [comment]
the same problem!

**Comment by USTBVEO** [comment]
您发给我的信件已收到。from Vien Yann/杨威
 
   This is an automatic reply, confirming that your e-mail was received. Thank you!

**Comment by UCASAlbert** [comment]
> same problem. I also used the windows os.
你好，贝壳校友。 the same problem.

**Comment by UCASAlbert** [comment]
> > Hi, I have the same issue as you. Have you figured it out?
> 
> I choose another github to train my own data to achieve classfication 3D model https://github.com/zhulf0804/Pointnet2.PyTorch hope this github can help you

hi, how many minutes dose the training process take? And what is your GPU?

**Comment by aruscha-k** [comment]
Had the same problem using wsl under windows.
Moved everything to a linux HPC and now I get the numbers mentioned in the repo/paper. This is weird though.

**Comment by USTBVEO** [comment]
您发给我的信件已收到。from Vien Yann/杨威
 
   This is an automatic reply, confirming that your e-mail was received. Thank you!

**Comment by 182425yang** [comment]
the same problem!
